bot/cogs/filtering.py

```diff
@@ -305,43 +289,51 @@ async def _filter_message(self, msg: Message, delta: Optional[int] = None) -> No
                             self.schedule_msg_delete(data)
                             log.trace(f"Offensive message {msg.id} will be deleted on {delete_date}")
 
-                        if is_private:
-                            channel_str = "via DM"
-                        else:
-                            channel_str = f"in {msg.channel.mention}"
+                        stats = self._add_stats(filter_name, match, msg.content)
+                        await self._send_log(filter_name, _filter["type"], msg, stats)
 
-                        message_content, additional_embeds, additional_embeds_msg = self._add_stats(
-                            filter_name, match, msg.content
-                        )
-
-                        message = (
-                            f"The {filter_name} {_filter['type']} was triggered "
-                            f"by **{msg.author}** "
-                            f"(`{msg.author.id}`) {channel_str} with [the "
-                            f"following message]({msg.jump_url}):\n\n"
-                            f"{message_content}"
-                        )
+                        break  # We don't want multiple filters to trigger
 
-                        log.debug(message)
-
-                        # Send pretty mod log embed to mod-alerts
-                        await self.mod_log.send_log_message(
-                            icon_url=Icons.filtering,
-                            colour=Colour(Colours.soft_red),
-                            title=f"{_filter['type'].title()} triggered!",
-                            text=message,
-                            thumbnail=msg.author.avatar_url_as(static_format="png"),
-                            channel_id=Channels.mod_alerts,
-                            ping_everyone=Filter.ping_everyone if not is_private else False,
-                            additional_embeds=additional_embeds,
-                            additional_embeds_msg=additional_embeds_msg
-                        )
+    async def _send_log(
+        self,
+        filter_name: str,
+        filter_type: str,
+        msg: discord.Message,
+        stats: Stats,
+        *,
+        is_eval: bool = False,
+    ) -> None:
+        """Send a mod log for a triggered filter."""
+        if msg.channel.type is discord.ChannelType.private:
+            channel_str = "via DM"
+            ping_everyone = False
+        else:
+            channel_str = f"in {msg.channel.mention}"
+            ping_everyone = Filter.ping_everyone
+
+        eval_msg = "using !eval" if is_eval else ""
+        message = (
+            f"The {filter_name} {filter_type} was triggered by {format_user(msg.author)} "
+            f"{channel_str} {eval_msg}with [the following message]({msg.jump_url}):\n\n"
```

Comment:
`eval_msg` should end with a space if the `is_eval` is set to `True`
Otherwise we will end with
`... via DM using !evalwith ...` as an example.